README.md

```diff
@@ -36,4 +36,4 @@ After completing installation type in your favourite shell:
 
 * [Practical Workflows for ES6 Modules, Fluent 2014](https://www.youtube.com/watch?v=0VUjM-jJf2U)
 * [Guy Bedford: Package Management for ES6 Modules, JSConf2014](https://www.youtube.com/watch?v=szJjsduHBQQ)
-* [Loader API specification](http://whatwg.github.io/loader/) 
```

Comment:
Sorry, I just used the github editor and I guess it wanted to add an extra space :-( Let me know if you want me to fix this.